operations/api_operation.go

```diff
@@ -0,0 +1,47 @@
+package operations
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+
+	"github.com/degica/barcelona-cli/utils"
+)
+
+type ApiOperationApiClient interface {
+	Request(method string, path string, body io.Reader) ([]byte, error)
+}
+
+type ApiOperation struct {
+	method string
+	path   string
+	body   *bytes.Buffer
+	client ApiOperationApiClient
+}
+
+func NewApiOperation(method string, path string, body *bytes.Buffer, client ApiOperationApiClient) *ApiOperation {
+	return &ApiOperation{
+		method: method,
+		path:   path,
+		body:   body,
+		client: client,
+	}
+}
+
+func (oper ApiOperation) run() *runResult {
```

Comment:
You use `oper` as an instance name of ApiOperation at some places, but I'd like to rename them to `apiOperation` if it's within the coding rule of golang.